test/bin/subscription_manager_register.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Enable subscription in the VM
+subscription_manager_register() {
+    local -r vmname="$1"
+
+    if [ -f /tmp/subscription-manager-org ]; then
+        # CI workflow
+        cat <<EOF > /tmp/sub.sh
```

Comment:
Should we use `mktemp` for all the temporary files?

Reply:
Good catch